ui/app/AppLayouts/Chat/panels/communities/CommunityChannelsAndCategoriesBannerPanel.qml

```diff
@@ -0,0 +1,100 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.13
+import QtGraphicalEffects 1.13
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Components 0.1
+
+import shared.panels 1.0
+import shared.status 1.0
+
+import utils 1.0
+
+Rectangle {
+    id: root
+    height: childrenRect.height + Style.current.padding
+    anchors.left: parent.left
+    anchors.leftMargin: Style.current.padding
+    anchors.right: parent.right
+    anchors.rightMargin: Style.current.padding
+    border.color: Style.current.border
+    radius: 16
+    color: Style.current.transparent
+    property string communityId
+
+
+   Rectangle {
+        width: 66
+        height: 4
+        color: Style.current.secondaryMenuBackground
+        anchors.top: parent.top
+        anchors.topMargin: -2
+        anchors.horizontalCenter: parent.horizontalCenter
+    }
+
+    StatusRoundIcon {
+        anchors.top: parent.top
+        anchors.topMargin: -6
+        anchors.horizontalCenter: parent.horizontalCenter
+        width: 40
+        height: 40
+        icon.name: "channel"
+    }
+
+    StatusFlatRoundButton {
```

Comment:
Wondering if using layouts instead of positioners and hardcoded sizes won't improve the visibility. Looks good otherwise.

Reply:
I am also always more for Layouts over anchoring or positioners since they are more convenient for UI.